.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: Build
+on: [ push, pull_request ]
+
+jobs:
+  build:
+    # Only run on PRs if the source branch is on someone else's repo
+    if: ${{ github.event_name != 'pull_request' || github.repository != github.event.pull_request.head.repo.full_name }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 8, 11, 17 ]
+      fail-fast: true
+    steps:
+      - uses: actions/checkout@v2.3.5
+      - uses: gradle/wrapper-validation-action@v1
+      - name: JDK ${{ matrix.java }}
+        uses: actions/setup-java@v2.3.1
+        with:
+          java-version: ${{ matrix.java }}
+          cache: 'gradle'
+          distribution: 'temurin'
```

Comment:
Never heard of that, is that kind of a typical Java "distribution" people use nowadays? It's fine for me, just want to make sure this is nothing very special or something :)

Reply:
Ah, looks like this is the replacement for AdoptOpenJDK. Clearly I'm not following Java development much anymore. :')

Reply:
Yes, [Eclipse Temurin](https://projects.eclipse.org/proposals/eclipse-temurin) is the continuation of the [AdoptOpenJDK](https://adoptopenjdk.net/) project. [Amazon Coretto](https://aws.amazon.com/corretto/) and [Microsoft's OpenJDK builds](https://www.microsoft.com/openjdk) are two other widely used OpenJDK distributions, they all are pretty much the same as far as just building a project though. I tend to go with Temurin as in the past I've used AdoptOpenJDK, and on a good amount of linux distros installing the "openjdk" package will default to that as well, although again there isn't really much difference.